src/mongosh/data/enumerators.json

```diff
@@ -1,93 +1,114 @@
-[{
-    "name": "Enumerations",
-    "status": "Active",
-    "people": {
-        "title": {
-            "Apprentice Candidate":"Learning any full-stack",
-            "Apprentice":"Completed full-stack, working through EngineerKit",
-            "Resident":"Completed EngineerKit, working within a speciality",
-            "Master":"Master Engineer within a speciality",
-            "Distinguished":"Recognized thought leader in a domain",
-            "N/A":""
+[
+    {
+        "name": "Enumerations",
+        "status": "Active",
+        "people": {
+            "title": {
+                "Candidate":"An Apprentice Candidate learning a full-stack",
+                "Apprentice": "A software engineer gaining broad skills",
+                "Resident": "A well-rounded software engineer with a grasp of technologies across all ITIL/ITSM domains and an area of Specialization",
+                "Master": "A senior software engineer recognized as a leader within their specialty",
+                "Distinguished": "A senior software engineer that has been recognized by their peers as a distinguished contributor",
+                "N/A": "Not a software engineer"
+            },
+            "status": {
+                "Pending": "Registered Online",
+                "Onboarding": "Introduction Call Scheduled",
+                "Matching": "Completed introduction, waiting on mentor match",
+                "Active": "Matched with a mentor and having regular encounters",
+                "Hold": "Member who was active, initiated pause in mentor work",
+                "Inactive": "Member who was active, has indicated that they are no long interested in mentorship services",
+                "TimedOut": "Member did not schedule introduction call within 2-weeks.",
+                "Archived": "Soft Delete Indicator"
+            },
+            "device": {
+                "Mac (Intel)": "Mac with Intel silicon",
+                "Mac (Apple)": "Mac with Apple sillicon",
+                "Linux": "Boots to Linux (dual boot or vm)",
+                "Windows": "Uses WSL for bash/linux"
+            },
+            "cadence": {
+                "As Needed": "Meetings are not fixed",
+                "Monthly": "Meetings are once a month",
+                "Every 2 Weeks": "Meetings are once every two weeks",
+                "Weekly": "Meetings are once every week",
+                "Twice a Week": "Meetings are twice every week",
+                "Daily": "Meetings are every day"
```

Comment:
These are fine, though you could also consider using the ISO 8601 interval format:
- Monthly: R/P1M
- Every 2 weeks: R/P2W
- Weekly: R/P1W
- Twice a week: R/P0.5W
- Daily: R/P1D

For more info:
https://en.wikipedia.org/wiki/ISO_8601#Repeating_intervals
https://resources.data.gov/schemas/dcat-us/v1.1/iso8601_guidance/#accrualperiodicity